torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -452,15 +452,23 @@ void CudaPrinter::visit(const AtomicAdd* v) {
 }
 
 void CudaPrinter::visit(const Max* v) {
-  os() << "maximum(";
+  if (is_integral(v->dtype().scalar_type())) {
+    os() << "maximum_int(";
```

Comment:
I think `std::max` is supported by nvrtc?  Maybe a mild preference for that if it is available, otherwise this is cool.

Reply:
No, but `max` and `min` are. Done.